stackdriver-errors.js

```diff
@@ -52,7 +52,10 @@ StackdriverErrorReporter.prototype.start = function(config) {
   this.projectId = config.projectId;
   this.targetUrl = config.targetUrl;
   this.context = config.context || {};
-  this.serviceContext = {service: config.service || 'web'};
+  this.serviceContext = {service: config.service || 'web',
+                         resourceType: config.resourceType || 'reported_errors',
+                         severity: 'ERROR' || config.severity
```

Comment:
Can you talk more about why you think severity should be stored in the config?

I would expect instead the severity to be provided every time an error is reported.